go.mod

```diff
@@ -24,4 +24,4 @@ require (
 	google.golang.org/api v0.22.0
 )
 
-go 1.13
+go 1.14
```

Comment:
Bumping this to 1.14 does not prevent it from building with Go 1.13. Go 1.13.9+ will build it just fine.

The behavior of Go modules is still being tweaked with every compiler release. This directive simply instructs the compiler to use the latest behavior. It's a good idea to keep it up-to-date.